crossbeam-epoch/src/internal.rs

```diff
@@ -376,7 +376,7 @@ pub struct Local {
 // https://github.com/crossbeam-rs/crossbeam/issues/551
 #[test]
 fn local_size() {
-    assert_eq!(2040, core::mem::size_of::<Local>());
+    assert!(core::mem::size_of::<Local>() <= 2040, "An allocation of `Local` should be <= 2048 bytes.");
```

Comment:
The message disagrees on the size - `2040` or `2048`?

Reply:
I "guess" the reason is memory allocator would have 8 bytes of overhead.  But it's just a guess, and I'd like to ping @stjepang ...

In any case, I'll improve the assert description.  It's confusing as you said.

Reply:
2040 is the actual size of the current `Local`, so when comparing using `<=`, I think `<= 2048` is fine.

Reply:
I remembered that it was actually me who added this test (#552). IIRC, I just reduced the number of objects and adjusted it to be less than 2048 bytes, so that 8 bytes has no particular meaning.

Reply:
It seems it's quite safe to make its size 2048: https://www.facebook.com/notes/facebook-engineering/scalable-memory-allocation-using-jemalloc/480222803919